### PR TITLE
minor cflite build integration docs changes.

### PR DESCRIPTION
**Pull request by oliverchang**
- Minor clarification about this using OSS-Fuzz's build chain. Be
  consistent about calling this part OSS-Fuzz (rather than
  ClusterFuzzLite), since that's what the image name says.

- Add project.yaml documentation. Related: #6196

- Refer to OSS-Fuzz's guide for efficient fuzzing.

### CODE REVIEW

File: docs/clusterfuzzlite/build_integration.md [hunk 1]
```diff
@@ -14,13 +14,20 @@ permalink: /clusterfuzzlite/build-integration/
...
-ClusterFuzzLite supports statically linked [libFuzzer targets]({{ site.baseurl }}/reference/glossary/#fuzz-target) on Linux.
+ClusterFuzzLite supports statically linked
+[libFuzzer targets]({{ site.baseurl }}/reference/glossary/#fuzz-target) on
+Linux.
+
+We re-use the [OSS-Fuzz](https://github.com/google/oss-fuzz) toolchain to make
+building easier. If you are familiar with this, most of the concepts here are
+exactly the same, with one key difference. Rather than checking out the source
+code in the [`Dockerfile`](#dockerfile) using `git clone`, the `Dockerfile`
+copies in the source code directly during `docker build`.
```
Comment (oliverchang) [inline docs/clusterfuzzlite/build_integration.md:25]: Is there anything else missing?

**Comment by oliverchang** [comment]
PTAL, just some small changes. We probably also want to think about making this more standalone so that it can live in its own repo.

File: docs/clusterfuzzlite/build_integration.md (hunk 1 above)
Comment (jonathanmetzman) [reply docs/clusterfuzzlite/build_integration.md:25]: I don't think so.
There's still the project.yaml difference which I haven't decided on yet. I think this is OK for now.

File: docs/clusterfuzzlite/build_integration.md [hunk 2]
```diff
@@ -33,10 +40,17 @@ Before you can start setting up your new project for fuzzing, you must do the fo
 - Clone the OSS-Fuzz repo: `git clone https://github.com/google/oss-fuzz.git`
 
 ## Generating an empty build integration
-Build integrations consist of two files, a `Dockerfile` that can install
-dependencies for your project, and a `build.sh` file that actually builds your
-project. These must live in the `.clusterfuzzlite` directory in the root of your
+
+Build integrations consist of three configuration files:
+* [./clusterfuzzlite/project.yaml](#projectyaml) - provides metadata about the project.
```
Comment (jonathanmetzman) [inline docs/clusterfuzzlite/build_integration.md:45]: Fine for now I guess.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: docs/clusterfuzzlite/build_integration.md (hunk 2 above)
Comment (oliverchang) [reply docs/clusterfuzzlite/build_integration.md:45]: Another reason to keep project.yaml: In the future we may want auto-bugfiling and specifying CCs (or github usernames). There's not really much of a cost apart there being an auto-generated file in most cases is there?

**Pull request merged**